src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class Application {
+    private static final Lottery lottery = new Lottery();
```

Comment:
Consider defining the number of balls as a constant to avoid magic numbers. This aligns with the checklist item: 'All magic numbers in your code should be constants.'